code/game/objects/structures/fence.dm

```diff
@@ -0,0 +1,161 @@
+//Chain link fences
+//Sprites ported from /VG/
+
+
+#define CUT_TIME 100
+#define CLIMB_TIME 150
+
+#define NO_HOLE 0 //section is intact
+#define MEDIUM_HOLE 1 //medium hole in the section - can climb through
+#define LARGE_HOLE 2 //large hole in the section - can walk through
+#define MAX_HOLE_SIZE LARGE_HOLE
+
+/obj/structure/fence
+	name = "fence"
+	desc = "A chain link fence. Not as effective as a wall, but generally it keeps people out."
+	density = 1
+	anchored = 1
+
+	icon = 'icons/obj/fence.dmi'
+	icon_state = "straight"
+
+	var/cuttable = TRUE
+	var/hole_size= NO_HOLE
+	var/invulnerable = FALSE
+
+/obj/structure/fence/New()
+	..()
+
+	update_cut_status()
+
+/obj/structure/fence/examine(mob/user)
+	.=..()
```

Comment:
``. =..()``

Reply:
No don't ruin the battleaxe of returning your parents.